tests/phpunit/integration/Modules/AdSenseTest.php

```diff
@@ -80,13 +75,8 @@ public function test_register_template_redirect_amp() {
 	}
 
 	public function test_register_template_redirect_non_amp() {
-		$context      = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
-		$mock_context = $this->getMockBuilder( 'MockClass' )->setMethods( array( 'is_amp', 'input' ) )->getMock();
-		$mock_context->method( 'input' )->will( $this->returnValue( $context->input() ) );
-		$mock_context->method( 'is_amp' )->will( $this->returnValue( false ) );
-
+		$context = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
```

Comment:
Maybe still worth introducing some kind of "no AMP" `Context` mock in `TestCase` because this here is somewhat implicit and ties into the real behavior of `Context` which could be considered somewhat error-prone. But nothing critical for now.